CHANGELOG.md

```
@@ -1,3 +1,33 @@
+## Version [Unreleased]
+
+## 💥 Breaking Changes
+
+**BREAKING CHANGE: Auth-Endpoints geben keine Tokens mehr im JSON-Body zurück**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)